Remotion/Web/Development.WebTesting/FluentControlSelection/IFluentControlSelector.cs

```diff
@@ -34,8 +33,8 @@ public interface IFluentControlSelector<out TControlSelector, TControlObject>
     /// </summary>
     /// <param name="selectionCommandBuilder">The selection command builder which is combined with the <see cref="IControlSelector"/>.</param>
     /// <returns>The <see cref="ControlObject"/> for the selected control.</returns>
-    /// <exception cref="AmbiguousException">If multiple matching controls are found but the specific <typeparamref name="TControlSelector"/> requires an exact match.</exception>
-    /// <exception cref="MissingHtmlException">If the control cannot be found.</exception>
+    /// <exception cref="WebTestException">If multiple matching controls are found but the specific <typeparamref name="TControlSelector"/> requires an exact match.</exception>

```

Comment:
We should probably merge this into a single Exception doc

Reply:
We found docs using this style.